ChangeLog.d/early-data.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Mbed TLS now supports the writing and reading of TLS 1.3 early data.
+     Early data is supported when the configuration option
+     MBEDTLS_SSL_EARLY_DATA is enabled (disabled by default).
```

Comment:
Perhaps also mention that it needs to be enabled at runtime using `mbedtls_ssl_conf_early_data()` as well?

Btw, looking at this function's documentation, it says `\warning This interface is experimental and may change without notice.` If that's the case we should mention in the ChangeLog that support is experimental. Otherwise we should remove the warning. (Edit: same for `MBEDTLS_SSL_EARLY_DATA`.)

Edit 2: OK, you removed the warning for the compile-time option, but the runtime function that people absolutely need to call in order to use this is still documented as experimental...

Reply:
> Btw, looking at this function's documentation, it says `\warning This interface is experimental and may change without notice.` If that's the case we should mention in the ChangeLog that support is experimental. Otherwise we should remove the warning. (Edit: same for `MBEDTLS_SSL_EARLY_DATA`.)

Thanks for catching that, I've fixed that (no experimental anymore) by amending the commit renamed "Remove experimental warnings related to early data".